tests/main.test.js

```diff
@@ -2,6 +2,8 @@ import request from 'supertest';
 
 import { ALL_PAGEABLE_ENDPOINTS, ALL_GEN_ENDPOINTS } from "./endpoints.js";
 
+var total_datapoints_per_table = 200;
```

Comment:
I made this global so I could use it to check correct page counts later. Also, I noticed that we were inconsistent in `datapoints` vs `data_points`, and since we used `max_datapoints` for the page limit tests, I switched this to `datapoints` as well.

Reply:
Cool, thx.